scripts/mcdo.sh

```diff
@@ -21,9 +21,9 @@ alias=$1 ; shift
 units=$1 ; shift
 vm=$1 ; shift
 
-tmp=$(mktemp -d -t climaf_mcdo_XXXXXX) # Will use TMPDIR if set, else /tmp
+tmp=$(mktemp -d -t $CLIMAF_CACHE/climaf_mcdo_XXXXXX) # Will use TMPDIR if set, else /tmp
```

Comment:
You can get the same behaviour (without your change) by setting 
  export TMPDIR=$CLIMAF_CACHE
before launching CliMAF, while you change would just mix up various use of CLIMAF_CACHE, which is not necessarily adequate for all cases (as e.g. a shared cache, or a cache on a less powerful file system ...)

Reply:
That makes sense. We can cancel this PR.